lib/src/Pos/Node/API.hs

```diff
@@ -605,12 +605,3 @@ type API =
         SettingsAPI
     :<|>
         InfoAPI
-    :<|>
```

Comment:
Removing this as it's wallet specific. More deleted code later in the PR shows why.

Reply:
It's actually the opposite. This API is truly node specific and has nothing to do with the wallet. Updates are unnecessarily stored in acid-state; a plain `MVar` would suffice actually.